cityofaustin/atd-moped
# Refine help menu navigation

## Associated issue
- https://github.com/cityofaustin/atd-data-tech/issues/14976

## steps to test
1. go to the [Deploy preview](https://deploy-preview-1214--atd-moped-main.netlify.app/) and click on all of the links on the navigation menu. 
2. make your browser window small enough so that the top bar navigation tabs collapse into the drop down. make sure the menu items all work correctly.

<img width="226" alt="Screenshot 2023-12-14 at 5 38 53 PM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/a24dca67-65db-4d8f-bb09-769073e0e363">
